addons/website/static/src/scss/survey.scss

```diff
@@ -0,0 +1,267 @@
+            overflow-y: auto;
+        }
+
+        .o_survey_logo_wrapper {
+            @extend %o-preview-alpha-background;
+            img {
+                object-fit: contain;
+            }
+        }
+
+        // ==== Screens
+
+        &.o_description_screen {
+            .dropdown, .industry_selection, .custom-ui-autocomplete {
+                font-weight: 500;
+                cursor: pointer;
+            }
+
+            .o_survey_type_dd.o_step_todo {
+                min-width: 13ch;
+            }
+
+            .industry_selection.o_step_todo {
+                min-width: 17ch;
+            }
+
+            .o_survey_purpose_dd.o_step_todo {
+                min-width: 12ch;
+            }
+
+            .dropdown {
+                box-shadow: inset 0 -3px 0 theme-color(primary);
+
+                &.o_step_todo .fa-angle-down {
+                    transform: translateY(-0.26em);
+                }
+
+                &.o_step_completed {
+                    transition: all 300ms ease;
+
+                    .fa-angle-down {
+                        @include o-position-absolute($right: 0);
+                        opacity:0;
+                    }
+
+                    &:hover, &.show {
+                        padding-right: 1.3rem!important;
+
+                        .fa-angle-down {
+                            transition: all 300ms ease 100ms;
+                            opacity:1;
+                        }
+                    }
+                }
+            }
+
+            .dropdown-menu, .custom-ui-autocomplete {
+                @include border-bottom-radius($border-radius);
+                font-size: inherit;
+            }
+
+            .dropdown-item, .fa-angle-down, .ui-menu-item a, .ui-menu-item a.ui-state-active {
+                font-size: .65em;
+                font-weight: inherit;
+            }
+
+            .industry_selection {
+                border: 1px solid $border-color;
+                color: theme-color(primary);
+
+                &:focus {
+                    border-color: theme-color(primary);
+                    color: inherit;
+                    outline: none;
+                }
+            }
+
+            .custom-ui-autocomplete {
+                max-height: 300px; // <- Fallback
+                max-height: 35vh;
+
+                .ui-menu-item {
+                    list-style: none;
+
+                    a {
+                        padding: $dropdown-item-padding-y*3 $dropdown-item-padding-x;
+                        font-family: $font-family-base;
+                        text-decoration: none;
+                        line-height: 1;
+
+                        &:hover, &.ui-state-active {
+                            background-color: $dropdown-link-hover-bg;
+                            color: $dropdown-link-color;
+                        }
+                    }
+                }   
+            }
+        }
+
+        &.o_logo_validation_screen {
```

Comment:
Not seeing this class anywhere else in the code. Maybe a leftover from a previous iteration?

Reply:
yep